osi_hostvehicledata.proto

```diff
@@ -2,31 +2,352 @@ syntax = "proto2";
+            optional uint32 index = 2;
+
+            // Dry friction is a force that opposes the relative lateral motion of two solid surfaces 
+            // in contact. It is subdivided into static friction between non-moving surfaces and kinetic
+            // friction between moving surfaces.
+            // Ued here is the dry friction coefficient of the paired materials (see reference).
+            // Dimensionless.
+            //
+            // Unit: -
+            //
+            // \par References:
+            // - http://adaptivemap.ma.psu.edu/websites/6_friction/dry_friction/dryfriction.html
+            //
+            //optional double kinetic_friction_coefficient = 3;
+
+            // Contact point of the mid of the tire tread with the world.
+            // Absolute coordinates (x, y, z).
+            //
+            // Unit: m
+            //
+            //optional Vector3d contact_point = 4;
+
+            // Contains the rotation rate of the wheel per second in radians.
+            // Based on the processed output of the hall sensor at the wheel.
+            //
+            // Unit: rad/s
+            //
+            optional double rotation_rate = 5;
+
+            // Contains the longitudinal slip of the tire.
+            // \par References:
+            // - https://www.kfz-tech.de/Biblio/Formelsammlung/Schlupf.htm
+            //
+            // Unit: %
+            //
+            optional double slip = 6;
+        }
+    }
+
+    //
+    // \brief This message contains all the information the vehicle knows about its kinematic states.
+    //
+    message VehiclePositionAndKinematics
+    {
+        //
+        // \brief Current kinematic data based on the output of the Intertial Measurement Unit (IMU).
+        //
+        message CartesianInformation
+        {
+            // Current kinematic data based on the output of the Intertial Measurement Unit (IMU).
+            //
+            // \note Note that dimension and base_polygon need not be set.
+            //
+            optional BaseMoving cartesian_data = 1;
+
+            // Current kinematic data error based on the output of the Intertial Measurement Unit (IMU).
+            //
+            // \note Note that dimension and base_polygon need not be set.
+            //
+            optional BaseMoving cartesian_data_rmse = 2;
+        }
+
+        //
+        // \brief Current estimated location based on the output of the GPS-Unit.
+        //
+        message GeoreferencedInformation
+        {
+            // Longitude in decimal degrees regarding WGS84.
+            //
+            // Unit: Degree
+            //
+            optional double longitude = 1;
+
+            // Latitude in decimal degrees regarding WGS84.
+            //
+            // Unit: Degree
+            //
+            optional double latitude = 2;
+
+            // Height above sea level regarding EGM96.
+            //
+            // Unit: m
+            //
```

Comment:
Range [-300;10000]

Reply:
Yes, it's really limited to these values?